README.md

```diff
@@ -1309,8 +1309,9 @@ app.UseSwaggerUI(c =>
     c.UseResponseInterceptor("(response) => { return response; }");
 });
 ```
- [!NOTE]
- When adding custom plugins, make sure you added custom `js` file that defines the plugin function.
+
+ > [!NOTE]
+ > When adding custom plugins, make sure you added custom `js` file that defines the plugin function(s).
```

Comment:
```suggestion
 > When adding custom plugins, make sure you add any custom `js` files that defines the plugin function(s).
```

Please accept the full suggestions, rather than just parts of them.

Reply:
Also, please don't self-resolve the review comments - I'll resolve them when I'm happy they're dealt with as requested.